### DISCUSSION TITLE
Joins?

### DISCUSSION THREAD

**Discussion by DakotaLarson** [prisma/prisma#5136] [Q&A]
Hi!
I was wondering if there is any way to supply joins to queries. Currently, supplying an "include" parameter to a query runs two queries, then joins the data together, as though it were a left join. Is it possible to change this behavior to use joins? 

Also, is it possible to only return results that have an included table? (an inner join essentially)

Thanks for any help. I am really impressed with Prisma so far.

**Comment by ryands17** [comment]
Hey @DakotaLarson 👋 

Currently Prisma has gone with the way of running separate queries to fetch relations, so unfortunately it isn't possible to change this behaviour to use `joins`. I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this :)

**Reply by DakotaLarson** [reply]
Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
